### ISSUE TITLE
Raspberry pi zero

### ISSUE THREAD

**Issue by MaybeRichard** [adamcohenhillel/ADeus#65]
Do I need to purchase an additional microphone module when deploying with Raspberry pi zero? I found that it doesn't seem to be stated in the documentation, and I'm confused about this

**Comment by makeiteasierapps** [comment]
Yes you will need this or any other mic that follows I2s standards 

https://www.amazon.com/Adafruit-I2S-MEMS-Microphone-Breakout/dp/B06XNL2GBW?crid=3DONFZH9HAKPU&keywords=adafruit+i2s+mems+microphone+breakout&qid=1707082816&sprefix=,aps,228&sr=8-3

**Comment by MaybeRichard** [comment]
Thank you so much

> Yes you will need this or any other mic that follows I2s standards
> 
> https://www.amazon.com/Adafruit-I2S-MEMS-Microphone-Breakout/dp/B06XNL2GBW?crid=3DONFZH9HAKPU&keywords=adafruit+i2s+mems+microphone+breakout&qid=1707082816&sprefix=,aps,228&sr=8-3
